docs/DOCUMENTATION.md

```diff
@@ -559,8 +559,7 @@ func main() {
 [Dev/P1337] Bob: 31 <nil>
 [Prod/P1337] John: 36 <nil>
 [Dev/P3] Pat: 39 <nil>
-Unable to locate record [record=[/P3] : 0, dbName=postgres]
-	record not found
+err != nil - true
```

Comment:
not useful please revert from consumption perspective

Reply:
So, this change in DOCUMENTATION.md is the result of a previous PR that got merged previously. For some reason, DOCUMENTATION.md was modified last time but didn't include this change.
Regarding the change itself, I wrote about it [here](https://github.com/vmware-labs/multi-tenant-persistence-for-saas/pull/51/files#r1275379889).